go.mod

```diff
@@ -78,6 +78,7 @@ require (
 	github.com/openzipkin/zipkin-go v0.2.2 // indirect
 	github.com/parnurzeal/gorequest v0.2.15
 	github.com/pborman/uuid v1.2.0
+	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
```

Comment:
It’s seem no need to import this package?

Reply:
Yep, this line add automatically to the go.mod file in my dev environment.

Reply:
> It’s seem no need to import this package?

Yep, this line add automatically to the go.mod file in my dev environment.